src/diffusers/pipelines/unclip/pipeline_unclip.py

```diff
@@ -185,6 +185,49 @@ def _encode_prompt(self, prompt, num_images_per_prompt, do_classifier_free_guida
 
         return text_embeddings, text_encoder_hidden_states, text_mask
 
+    def enable_sequential_cpu_offload(self, gpu_id=0):
+        r"""
+        Offloads all models to CPU using accelerate, significantly reducing memory usage. When called, the pipeline's
+        models have their state dicts saved to CPU and then are moved to a `torch.device('meta') and loaded to GPU only
+        when their specific submodule has its `forward` method called.
+        """
+        if is_accelerate_available():
+            from accelerate import cpu_offload
+        else:
+            raise ImportError("Please install accelerate via `pip install accelerate`")
+
+        device = torch.device(f"cuda:{gpu_id}")
+
+        # TODO: self.prior.post_process_latents is not covered by the offload hooks, so it fails if added to the list
+        models = [
+            self.decoder,
+            self.text_proj,
+            self.text_encoder,
+            self.super_res_first,
+            self.super_res_last,
+        ]
```

Comment:
Not adding `self.prior` because `prior.post_process_latents` is not registered by the accelerate hooks (this is expected, only `forward` is supported)